client/src/views/ListFlyingSites.vue

```diff
@@ -1,6 +1,20 @@
 <template>
   <div v-if="sites" class="container-lg">
     <h3>Fluggebietsübersicht</h3>
+    <p>
+      Der XCCup ist ein mitteldeutscher Flachland Wettbewerb. Flüge können dabei
+      nur von registrierten Startplätzen eingereicht werden.
+    </p>
+    <p>
+      Startplätze können von allen im XCCup gemeldeten Personen vorgeschlagen
+      werden. Diese müssen dabei einen
+      Flachland-/Mittelgebirgscharakter aufweisen und im örtlichen Zusammenhang
```

Comment:
Ich dachte Mittelgebirge wollten wir nicht?

Reply:
Aber in welchen unserer aktuellen Regionen gibt es wirklich ein Flachland?
Es ist doch eigentlich alles Mittelgebirge oder?
"Richtiges" Flachland wäre dann wirklich so in Richtung Norden und Osten.

Deshalb hatte ich dann hier diesen "Kombi"-Charakter gewählt.